### DISCUSSION TITLE
[BUG] ive deleted wallet db and nooits just loading like this forever, and do i need to delete the debug file in db as well?

### DISCUSSION THREAD

**Discussion by gnikhil002** [Chia-Network/chia-blockchain#9235] [Support]
**Describe the bug**

**To Reproduce**

![chia glich 2](https://user-images.githubusercontent.com/93718968/140466031-a3ae5ff9-da85-45f3-a12f-796f88894382.PNG)

**Expected behavior**

**Screenshots**

**Desktop**

- OS: 
- OS Version/Flavor: 
- CPU: 

**Additional context**

**Comment by loppefaaret** [comment]
without knowing what OS you are on, and what version of Chia - you can always try and move the main blockchain database out, and see if that doesnt get it started syncing from 0:
WINDOWS BLOCKCHAIN DATABASE RESET

Close down the Chia app, and open up a PowerShell window, and execute the below command:
```
mv ~\.chia\mainnet\db\block* ~\Desktop\
```
Then try opening up the Chia app again
If it worked, you can remove the files we put on the Desktop!

adjust for other OS if needed

**Comment by samches1337** [comment]
Same problem here after update using your guide.
OS: Ubuntu 20.04 LTS
last time sync took me more than a month... 

tried to delete all in db and wallet db

**Comment by loppefaaret** [comment]
getting the databases onto faster storage might be something worth looking into then - SD cards and mechanical HDD's arent the best at handling the random IO our database demands
